### ISSUE TITLE
"exit with reason killed in context child_terminated" problem

### ISSUE THREAD

**Issue by xsir99** [seth/pooler#69]
I don't know why, I get the following error every one or two mins.
```
15:51:30.964 [error] Supervisor pooler_pgsql_pool_member_sup had child epgsql started with {epgsql,connect,undefined} at <0.25555.8> exit with reason killed in context child_terminated
15:51:30.965 [error] Supervisor pooler_pgsql_pool_member_sup had child epgsql started with {epgsql,connect,undefined} at <0.9309.8> exit with reason killed in context child_terminated
15:51:30.965 [error] Supervisor pooler_pgsql_pool_member_sup had child epgsql started with {epgsql,connect,undefined} at <0.31360.7> exit with reason killed in context child_terminated
15:51:30.966 [error] Supervisor pooler_pgsql_pool_member_sup had child epgsql started with {epgsql,connect,undefined} at <0.9305.8> exit with reason killed in context child_terminated
15:51:30.966 [error] Supervisor pooler_pgsql_pool_member_sup had child epgsql started with {epgsql,connect,undefined} at <0.10522.7> exit with reason killed in context child_terminated
15:51:30.966 [error] Supervisor pooler_pgsql_pool_member_sup had child epgsql started with {epgsql,connect,undefined} at <0.25559.8> exit with reason killed in context child_terminated
15:51:30.967 [error] Supervisor pooler_pgsql_pool_member_sup had child epgsql started with {epgsql,connect,undefined} at <0.17923.7> exit with reason killed in context child_terminated
15:51:30.969 [error] Supervisor pooler_pgsql_pool_member_sup had child epgsql started with {epgsql,connect,undefined} at <0.18451.9> exit with reason killed in context child_terminated
15:52:30.965 [error] Supervisor pooler_pgsql_pool_member_sup had child epgsql started with {epgsql,connect,undefined} at <0.18430.9> exit with reason killed in context child_terminated
15:52:30.965 [error] Supervisor pooler_pgsql_pool_member_sup had child epgsql started with {epgsql,connect,undefined} at <0.18448.9> exit with reason killed in context child_terminated

```

Does anyone help me? very thanks!!

**Comment by sata** [comment]
Which version of Pooler are you using? What is your pooler configuration? 

most likely it's the same problem as in issue #60 which is solved by #66

**Comment by xsir99** [comment]
@sata Yes, it's version of the problem, Thinks!
